docs/articles/documentation/using-testcafe/command-line-interface.md

```diff
@@ -0,0 +1,226 @@
+---
+layout: docs
+title: Command Line Interface
+permalink: /documentation/using-testcafe/command-line-interface/
+---
+# Command Line Interface
+
+TestCafe command line interface is pretty simple.
+It consists of a single command that runs tests with a number of options.
+
+## Synopsis
+
+```sh
+testcafe [options] <browser-list-comma-separated> <file-or-glob ...>
+```
+
+## Browser List
+
+### Local Browsers
+
+Specify the list of browsers (separated by commas) where tests will be ran.
+You can specify browser paths or aliases. 
+Aliases are short names TestCafe assigns to browsers for quick reference. 
+A list of all the available aliases can be obtained by the `testcafe --list-browsers` command (see [below](#b)).
```

Comment:
> (see [below](#b))

This is ain't cool. Maybe we can rephrase is it, so we will have inline link, e.g.:
```
A list of all the available aliases can be obtained by the [--list-browsers](#b) command.
```